Python Examples/TestMonitor/test_data_manager_client.py

```diff
@@ -0,0 +1,183 @@
+
+    return request_response.json()
+
+def update_results(results: List) -> Dict:
+    """
+    Updates existing test results by merging or replacing values.
+    :param results: List of results to be updated
+    :return: json response after updating the results
+    """
+    if len(results) == 0 :
+        raise ValueError("Number of results to be updated can not be empty.")
+    body = update_test_results_request(results, determine_status_from_steps=True)
+    request_url = base_url + update_results_route
+    request_response = raise_post_request(request_url, body)
+
+    return request_response.json()
+
+def create_steps(steps: List) -> Dict:
+    """
+    Creates new test steps from the supplied models. 
+    The result associated with the steps must exist prior to step creation. 
+    The server automatically generates step IDs if not provided.
+    :param steps: Steps to be created
+    :return: json response after creating steps
+    """
+    if len(steps) == 0 :
+        raise ValueError("Number of steps to be created can not be empty.")
+    body = test_step_create_or_update_request_object(
+            steps, update_result_total_time=True
+        )
+    request_url = base_url + create_steps_route
+    request_response = raise_post_request(request_url, body)
+
+    return request_response.json()
+
+def update_steps(steps: List) -> Dict:
+    """
+    Updates existing steps by merging or replacing values.
+    :param steps: List of steps to be updated
+    :return: json response after updating steps
+    """
+    if len(steps) == 0 :
+        raise ValueError("Number of steps to be updated can not be empty.")
+    body = test_step_create_or_update_request_object(
+                steps, update_result_total_time=True
+            )
+    request_url = base_url + update_steps_route
+    request_response = raise_post_request(request_url, body)
+
+    return request_response.json()
+
+def delete_result(result_id: str, delete_steps: bool = True) -> None:
+    """
+    Deletes test result.
+    :param result_id: result ID to be deleted
+    """
+    if result_id == None :
+        raise ValueError("Missing required parameter 'result_id' in ResultsApi->DeleteResultV2 request.")
+    request_url = f"{base_url}{delete_result_route}/{result_id}?deleteSteps{delete_steps}"
+    raise_delete_request(request_url)
+
+def delete_results(result_ids: List, delete_steps: bool = True) -> Dict:
+    """
+    Delete multiple test results.
+    :param result_ids: List of result IDs to be deleted
+    :return: json response after deleting the results
+    """
+    if result_ids == None or len(result_ids) == 0:
+        raise ValueError("result_ids is a required property for ResultsApi->DeleteResultsV2 and cannot be null or empty.")
+    request_url = f"{base_url}{delete_results_route}"
+    body = delete_results_request(result_ids, delete_steps)   
+    request_response = raise_post_request(request_url, body)
+    if request_response.status_code != 204 :
```

Comment:
What about a partial success response for Delete, can we handle it as we did in Create Examples?